metrics/report/README.md

```diff
@@ -0,0 +1,48 @@
+* [Kata Containers metrics report generator](#kata-containers-metrics-report-generator)
+   * [Data gathering](#data-gathering)
+   * [Report generation](#report-generation)
+
+# Kata Containers metrics report generator
+
+The files within this directory can be used to generate a 'metrics report'
+for Kata Containers.
+
+The primary workflow consists of two stages:
+
+1) Run the provided report metrics data gathering scripts on the system(s) you wish
+to analyse.
+2) Run the provided report generation script to analyse the data and generate a
+report file.
+
+## Data gathering
+
+Data gathering is provided by the `grabdata.sh` script. When run, this script will
+execute a set of tests from the `tests/metrics` directory. The resulting JSON results files
+will be placed into the `tests/metrics/results` directory.
+
+Once the results have been generated, create a suitably named subdirectory of
+`tests/metrics/results`, and move the JSON files into it.
+
+Repeat this process if you wish to compare multiple sets of results. Note, the
+report generation scripts will process all subfolders of `tests/metrics/results` when
+generating the report.
+
+## Report generation
+
+Report generation is provided by the `makereport.sh` script. By default this script will
```

Comment:
Lines 32-33 suggested rewrite to stay active: ...By default, this script processes all subdirs...